numpyro/distributions/continuous.py

```diff
@@ -1680,38 +1680,46 @@ def infer_shapes(df, loc, scale_tril):
         return batch_shape, event_shape
 
 
-class MultivariateBeta(Distribution):
+class GaussianCopula(Distribution):
     arg_constraints = {
-        "concentration1": constraints.positive,
-        "concentration0": constraints.positive,
+        "covariance_matrix": constraints.positive_definite,
+        "precision_matrix": constraints.positive_definite,
+        "scale_tril": constraints.lower_cholesky,
     }
-    reparametrized_params = ["concentration1", "concentration0"]
+    support = constraints.real_vector
```

Comment:
You can use
```python
@property
def support(self):
    return constraints.independent(self.transform_dist.support, 1)
```

Reply:
I see the following decorator used with `support` used elsewhere. Would you recommend using it? I suppose we don't necessarily know the event dim?

```python
@constraints.dependent_property(is_discrete=False, event_dim=0)
def support(self):
    ...
```

Reply:
Wouldn't the support be static `constraints.independent(constraints.unit_interval, 1)`?  Sorry if I misunderstand, but if `Beta.support == constraints.unit_interval`, I'd expect this distribution to have static support.

> I suppose we don't necessarily know the event dim?

IIUC we know `event_dim == 1`, so `event_shape == (n,)` for some `n` we don't know, but constraints don't need to know that `n`.

Sorry if I'm misunderstanding something!

Reply:
@fritzo Thanks for the comment. You are right `event_dim` is known even though `event_shape` is not.